docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -76,20 +74,164 @@ The bundling strategy was taken care of by a Create React App, and no locale was
+  ```
+
+- `LocalizationProvider` is exported by both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`:
+
+  ```tsx
+  import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+  import { LocalizationProvider } from '@mui/x-date-pickers';
+  import { LocalizationProvider } from '@mui/x-date-pickers-pro/LocalizationProvider';
+  import { LocalizationProvider } from '@mui/x-date-pickers-pro/';
+  ```
+
+:::
+
 :::info
 If you need to use the Date and Time Pickers with a custom locale, have a look at the [Localized dates](/x/react-date-pickers/adapters-locale/) page.
 :::
 
+## Commonly used components
+
+Most applications only need a few of the components exposed by the Date and Time packages:
+
+{{"demo": "CommonlyUsedComponents.js"}}
+
+:::success
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but not with nested imports.
+
+For example, to use the `DatePicker` component, the three following imports are valid:
+
+```tsx
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { DatePicker } from '@mui/x-date-pickers';
+import { DatePicker } from '@mui/x-date-pickers-pro';
+```
+
+:::
+
+## Basic use cases
+
+### Controlled value
+
+All the components can be controlled using the `value` / `onChange` props:
+
+{{"demo": "ControlledComponent.js"}}
+
+### Custom format
+
+All the components with an HTML input can receive a custom format using the [`format` prop](/x/react-date-pickers/adapters-locale/#custom-field-format):
+
+{{"demo": "CustomFormat.js"}}
+
+### Form props
+
+All the components component can be disabled or read-only:
+
+{{"demo": "FormProps.js"}}
+
+### Validation
+
+If you need to add validation to your component, you can check the [Validation page](/x/react-date-pickers/validation/):
+
+{{"demo": "ValidatedComponent.js"}}
+
+## Other components
+
+### Keyboard or mouse editing
+
+The first thing you want to decide, is how the user should be able to select their value:
+
+- For keyboard and mouse editing, use the _Picker_ components:
+
+{{"demo": "BasicDatePicker.js", "hideToolbar": true, "bg": "inline"}}
+
+- For keyboard-only editing, use the _Field_ components:
+
+{{"demo": "BasicDateField.js", "hideToolbar": true, "bg": "inline"}}
+
+- For mouse-only editing, use the _Calendar / Clock_ components:
+
+{{"demo": "BasicDateCalendar.js", "hideToolbar": true, "bg": "inline"}}
+
+:::success
+Each _Picker_ is the combination of one _Field_ and one or several _Calendar / Clock_ components.
+For example, the `DatePicker` is the combination of the `DateField` and the `DateCalendar`.
+
+The _Calendar / Clock_ components are rendered inside a _Popover_ on desktop and inside a _Modal_ on mobile.
+:::
+
+### Date or time editing ?
+
+The Date and Time Pickers are divided into six families of components.
+The demo below shows each one of them using their field component:
+
+{{"demo": "ComponentFamilies.js"}}
+
+### Responsiveness
```

Comment:
Hm...
This would create an inconsistency between our new approach of naming such section in individual docs as `Available components`.

Reply:
True, but if we use `Available components` it's super unclear in the context of this page.

On the `date-picker.md`, `Available components` list all the components in the scope of the page.
That's not the case here, we are just talking about the component dedicated to the various type of device.

If at one point we migration to the modal vs popover naming, then we could have a name describing this behavior.

Reply:
Yes, you could skip them. Their main use case is if you need to create advanced components where you override the Modal/Popper. So it can be deeper in the docs

Reply:
If you remove entirely this section, would you have no place to describe our responsive behaviors outside of the component pages ? 

That can make sense
Or we could, in the future, have a page describing more in depth how we handle responsiveness (a boosted version of the `Available components` sections but with examples using `desktopMediaQuery`)